lib/db.js

```diff
@@ -0,0 +1,378 @@
+export const contacts = [
+  {
+    id: "1",
+    name: "Carmen River",
+    phone: "+123456781",
+    note: "Gute Freundin",
+    photo:
+      "https://images.unsplash.com/photo-1534528741775-53994a69daeb?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=1528&q=80",
+    messages: [
+      "😃",
+      "Hey! Ich vermisse dich so sehr!",
+      "Lass uns bald wieder treffen und quatschen",
+      "Hallo Liebling! ",
+      "...hatte gerade einen lustigen Moment und musste an dich denken",
+    ],
+  },
+  {
+    id: "2",
+    name: "Esther Habib",
+    phone: "+123456782",
+    note: "Cheffin",
+    photo:
+      "https://images.unsplash.com/photo-1531123897727-8f129e1688ce?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8OHx8cG9ydHJhaXR8ZW58MHx8MHx8fDI%3D&auto=format&fit=crop&w=500&q=60",
+    messages: [
+      "🚀",
+      "Danke für euren großartigen Einsatz! Ihr seid ein fantastisches Team",
+      "Danke für eure harte Arbeit!",
+      "Ich bin stolz auf das, was wir gemeinsam erreichen",
+      "Euer Engagement ist bemerkenswert",
+    ],
+  },
+  {
+    id: "3",
+    name: "Jakob Lopez",
+    phone: "+123456783",
+    note: "Arbeitskolleg",
+    photo:
+      "https://images.unsplash.com/photo-1542327897-d73f4005b533?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8NTV8fHBvcnRyYWl0fGVufDB8fDB8fHwy&auto=format&fit=crop&w=500&q=60",
+    messages: [
+      "⭐️",
+      "wir sind fertig, yuhuuuu",
+      "war schwer, aber wir haben es geschafft",
+      "Hallo! Deine Professionalität und Effizienz sind inspirierend. Weiter so!",
+      "Danke für deine Unterstützung und Zusammenarbeit. Wir sind ein starkes Team!",
+    ],
+  },
+  {
+    id: "4",
+    name: "Lola Fischer",
+    phone: "+123456784",
+    note: "Nachbarin",
+    photo:
+      "https://images.unsplash.com/photo-1597954088261-4dc20374af14?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=387&q=80",
+    messages: [
+      "🌼",
+      "du hast echt einen grünen Daumen",
+      "Falls du jemals eine Tasse Zucker oder ein fehlendes Ei für ein Rezept benötigst, komm einfach vorbei",
+      "Wir helfen uns gerne gegenseitig aus",
+      "... es mir leid tut, wenn der Lärm gestern Abend gestört hat",
+    ],
+  },
+  {
+    id: "5",
+    name: "Manuel Acar",
+    phone: "+123456785",
+    note: "Gute Freund",
+    photo:
+      "https://images.unsplash.com/photo-1504810370725-2585de12e6ee?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=387&q=80",
+    messages: [
+      "🥳",
+      "Denk daran, dass ich dich unterstütze",
+      "Ich bin so dankbar",
+      "Ich bewundere deine Entschlossenheit und deinen Mut",
+      "Fühl dich gedrückt!",
+    ],
+  },
+];
+//   {
+//     id: "6",
```

Comment:
You maybe should remove the commented-out objects, or?

Reply:
i do it